CONTRIBUTING.md

```diff
@@ -356,8 +356,8 @@ format.
 ---
 synopsis: Add feature xyz
 packages: [cabal-install]
-prs: #0000
-issues: #0000 #0000
+prs: 0000
+issues: 0000 0000
```

Comment:
```suggestion
issues: [0000, 0000]
```

The `issues` field (unlike the `prs` field!) is a list.

Reply:
@ffaf1, is all this correct? I recall you claiming it wasn't and that it would be a breaking change to make it so.

Reply:
The `issues` and `prs` yaml fields are flexible: they can be lists of numbers or strings containing whitespace-separated numbers ([implementation](https://codeberg.org/fgaz/changelog-d/src/commit/0120b619ab3e4827914234674c0a3c7009642901/src/ChangelogD.hs#L454-L466)). So any of the following is correct:

```yaml
issues: 1 2 3
```
```yaml
prs: 1 2 3
```
```yaml
issues:
  - 1
  - 2
  - 3
```
```yaml
prs:
  - 1
  - 2
  - 3
```

The example in README.md was supposed to show that you can do both, but I see it ended up just causing confusion.

Reply:
Sorry, pinged the wrong person.